packages/central-server/src/apiV2/surveyResponses/GETSurveyResponses.js

```diff
@@ -20,6 +20,11 @@ import { getQueryOptionsForColumns } from '../GETHandler/helpers';
 export class GETSurveyResponses extends GETHandler {
   permissionsFilteredInternally = true;
 
+  customJoinConditions = {
+    entity: ['entity.id', 'survey_response.entity_id'],
+    country: ['country.code', 'entity.country_code'],
+  };
+
   async findSingleRecord(surveyResponseId, options) {
```

Comment:
@EMcQ-BES can you cast your eyes over this one?

Reply:
I'm still getting this error:
![image](https://github.com/beyondessential/tupaia/assets/129009580/97acaf32-2d55-4c43-abeb-087650165268)